src/buildApplication/buildApplication.ts

```diff
@@ -115,28 +119,39 @@ async function prepareAndBuildWorker(
 	);
 }
 
-function printInvalidFunctionsErrorMessage(invalidFunctions: string[]): void {
+async function printInvalidFunctionsErrorMessage(
+	invalidFunctions: string[]
+): Promise<void> {
+	const nextVersion = coerce(await getPackageVersion('next'));
+
+	const { exportText, exampleCode } =
+		!nextVersion || versionGreaterThan(nextVersion, '13.1.2')
+			? {
+					exportText: 'the following edge runtime route segment config',
+					exampleCode: "export const runtime = 'edge';",
+			  }
+			: {
+					exportText: 'a config object specifying the edge runtime, like',
+					exampleCode: "export const config = { runtime: 'edge' };",
+			  };
+
+	const invalidRoutes = Array.from(
+		new Set(invalidFunctions.map(fn => fn.replace(/(\.rsc)?\.func$/, '')))
+	);
+
 	cliError(
 		`
 		ERROR: Failed to produce a Cloudflare Pages build from the project.
 
-			The following functions were not configured to run with the Edge Runtime:\n${invalidFunctions
-				.map(fn => `				- ${fn}`)
+			The following routes were not configured to run with the Edge Runtime:\n${invalidRoutes
+				.map(route => `			  - ${route}`)
 				.join('\n')}
 
-			If this is a Next.js project:
```

Comment:
Currently (and for the foreseeable future) the adaptor only supports `Next.js` so I don't really think that we should include this (it just adds confusion, besides adding it back when/if we'll support other frameworks should be rather trivial)